src/entries/popup/pages/settings/customChain/index.tsx

```diff
@@ -92,18 +93,29 @@ const KNOWN_NETWORKS: { name: string; networkInfo: customNetworkInfo }[] = [
     },
   },
   {
-    name: 'Blast Testnet',
+    name: 'Aurora Testnet',
```

Comment:
would it be a good moment to try to do a loop through viem/chains if is possible, so we don't add more hardcoded code like this?

Reply:
@estebanmino I think we'll be able to pull these from the Backend soon. I'm sharing these changes with Benjamin so that he can see what we hardcode currently